spec/tasks/review_coverage_backfill_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'rake'
+
+# rubocop:disable RSpec/DescribeClass
```

Comment:
I way to find to avoid this is the task only calls another file  review_coverage.rb
 that have all the logic. feel free to ignore